### PR TITLE
Boyscout-version-12-bump

### PR DESCRIPTION
**Pull request by Gecete**

### CODE REVIEW

**Review by wwjhu — CHANGES_REQUESTED** [review]
Doing this needs to wait till we're about to release the feature. `test/resources/minimal-sample-1_11_x/components-definition.json` needs to contain the new features we'll add and we're not done yet.

**Comment by Gecete** [comment]
Ok so, we can test it in the meantime by using `-next` tag once the other PR is merge, so if it detects that the schema version is -next it will pick the latest changes under development, but officially no new version is released yet

**Pull request closed without merging**
